src/main/interface/msp.c

```diff
@@ -1033,9 +1033,10 @@ static bool mspProcessOutCommand(uint8_t cmdMSP, sbuf_t *dst)
         sbufWriteU8(dst, gpsSol.numSat);
         sbufWriteU32(dst, gpsSol.llh.lat);
         sbufWriteU32(dst, gpsSol.llh.lon);
-        sbufWriteU16(dst, gpsSol.llh.alt);
+        sbufWriteU16(dst, MIN(gpsSol.llh.alt / 10, 65535));
         sbufWriteU16(dst, gpsSol.groundSpeed);
         sbufWriteU16(dst, gpsSol.groundCourse);
+        sbufWriteU32(dst, gpsSol.llh.alt);
```

Comment:
Is this necessary? 16bit range in 0.1m may be enough ...

Reply:
I agree - the extra 32bit alt isn't used in msp